_posts/2021-02-19-26-rln-relay.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+name:  "Privacy-preserving economic spam protection in Waku v2 with Rate Limiting Nullfiers"
+title:  "Privacy-preserving economic spam protection in Waku v2 with Rate Limiting Nullfiers"
+date:   2021-02-25 12:00:00 +0800
+author: staheri14
+published: true
+permalink: /Waku-v2-RLN-Relay
+categories: research
+summary: This post is going to give you an overview of how spam protection can be achieved in Waku Relay through rate-limiting nullifiers. We will cover a summary of spam-protection methods in centralized and p2p systems, and the solution overview and details of the economic spam-protection method. The open issues and future steps are discussed in the end.
```

Comment:
Be careful with preview length here. I think there's risk only part of this will be shown in preview for e.g. Twitter. Can check with https://cards-dev.twitter.com/validator and sample post. I will also sometimes post it to site and check Twitter card preview before publsihing it further

Reply:
Sure, nice idea!